### PR TITLE
Add search form to 404 page

### LINKED ISSUE #1305: Add search form to 404 page
From https://github.com/flarum/core/commit/ea2fc1ff8a7972fd809c9b45b552bda3138ed55e#commitcomment-26252028

### PR DESCRIPTION
**Pull request by lsilvio**
**Fixes flarum/framework#1305 **

**Changes proposed in this pull request:**
Implement search form on 404 page.

**Screenshot**
image

**Confirmed**
- [x] Frontend changes: tested on a local Flarum installation.

### CODE REVIEW

File: views/error/not_found.blade.php [hunk 1]
```diff
@@ -4,6 +4,12 @@
   <p>
     {{ $message }}
   </p>
+  <p>
+    <form class="form" method="GET" action="/">
```
Comment (datitisev) [inline views/error/not_found.blade.php]: We cannot use `/` here for the action as it won't work with subfolders.
And is there a reason why the `form` has a class? It's not used anywhere, and styles only come from the inline `style` tag.

```suggestion
    <form class="form" method="GET" action="{{ $app->url() }}">
```

**Review by dsevillamartin — CHANGES_REQUESTED** [review]

**Comment by datitisev** [comment]
Also - the button has styles that aren't shown in your screenshot... and that mess up the styling.

image

**Comment by luceos** [comment]
We probably should just merge this and fix the last few things on this, anyone up to tackle this?

**Comment by askvortsov1** [comment]
While this solves the issue, I would propose that we consider an alternative approach, such as https://github.com/flarum/core/issues/654#issuecomment-593487567.

**Comment by askvortsov1** [comment]
Considering a lack of activity and consensus on where exactly we want to take this, I'm going to close this PR for now (can always reopen later) as we already have a lot open, and this isn't a priority for beta 14. Discussion can continue in flarum/framework#1305 and flarum/issue-archive#377

**Pull request closed without merging**